cmd/cli/serve/util.go

```diff
@@ -152,6 +150,15 @@ func SetupIPFSClient(ctx context.Context, cm *system.CleanupManager, ipfsCfg typ
 		return ipfs.Client{}, fmt.Errorf("error creating IPFS client: %s", err)
 	}
 
+	if len(ipfsCfg.SwarmAddresses) != 0 {
+		maddrs, err := ipfs.ParsePeersString(ipfsCfg.SwarmAddresses)
+		if err != nil {
+			return ipfs.Client{}, err
+		}
+
+		client.SwarmConnect(ctx, maddrs)
+	}
+
```

Comment:
Instructs the remote ipfs client to connect to swarm peers specified by flag, envvar or config.